user_tools/src/spark_rapids_tools/tools/unsupported_ops_stage_duration.py

```diff
@@ -29,17 +29,17 @@ class UnsupportedOpsStageDuration:
     """
     props: dict = field(default=None, init=True)
 
-    def prepare_apps_with_unsupported_stages(self, all_apps: pd.DataFrame,
+    def prepare_apps_with_unsupported_stages(self, processed_apps: pd.DataFrame,
```

Comment:
If the argument name is changed to reflect better description of how it is used, then perhaps adding a method comment to describe the argument would be helpful for everyone working on that method moving fwd.